R/checkAggrBeforeResample.R

```diff
@@ -0,0 +1,25 @@
+# check whether rdesc$predict is set, so that the requiring properties of the measure are satisfied
+# called the beginning of resample
+checkAggrBeforeResample = function(measure, rdesc) {
+  a = measure$aggr
+  p = a$properties
+  pred = rdesc$predict
+  p.allowed = if (all(c("req.train", "req.test") %in% p)) {
```

Comment:
> Why not setequal here?

because my code expresses what i want to say, and is more robust them setequal. consider that the aggregation properties might grow larger. and contain other stuff.